src/app/styles/partials/_base.scss

```diff
@@ -19,13 +19,14 @@ $z-index-bottom: 0;
 $profile-pic-height: 185px;
 $profile-pic-width: 200px;
 $profile-pic-padding: $padding-large;
+$peace-corps-left-banner-height: 1.6em;
```

Comment:
can this be changed to pixels, just for consistency. Not a big deal otherwise.

Reply:
Actually, Vasitha had asked me to change it to 'em' from pixels :sweat_smile: @shreyans29 what to do?

Reply:
okay, no don't change it then.